iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractMergeNetworkTest.java

```diff
@@ -127,10 +127,10 @@ public void testMergeAndDetach() {
         DanglingLine dl1 = detachedN1.getDanglingLine("dl1");
         DanglingLine dl2 = merge.getDanglingLine("dl2");
         // - P0 and Q0 of the removed tie line's underlying dangling lines were updated:
-        assertEquals(-1724.437, dl1.getP0(), 0.001);
-        assertEquals(1605.281, dl1.getQ0(), 0.001);
-        assertEquals(-1724.437, dl2.getP0(), 0.001);
-        assertEquals(1605.281, dl2.getQ0(), 0.001);
+        assertEquals(-731.312, dl1.getP0(), 0.001);
+        assertEquals(-1254.625, dl1.getQ0(), 0.001);
+        assertEquals(-731.312, dl2.getP0(), 0.001);
+        assertEquals(-1254.625, dl2.getQ0(), 0.001);
```

Comment:
note: this test case uses a dummy network with _huge_ B and G shunt values for dangling lines (G = 4.0 Siemens and B = 5.0 Siemens) hence the large diff in results. The values used are not representative of what exists in real life which would be rather in the range of _eventually up to_ a milli-Siemens for _very_ long overhead lines or cables.